sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -785,6 +786,182 @@
         }
     }
 
+    /**
+     * Analyze ID documents using optical character recognition (OCR) and a prebuilt ID trained
```

Comment:
"prebuilt ID trained model" seems a little awkward. "and a prebuilt model trained on ID documents"?